app-guides/mysql-on-fly.html.markerb

```diff
@@ -97,15 +96,21 @@ There are a few important things to note:
 1. In the `[env]` section, we added two not-so-secret environment variables that MySQL will need to initialize itself.
     * The `MYSQL_USER` here should be any user but `root`, which already exists.
 1. We added the `[processes]` section for the default `app` process, which lets us pass custom commands (overriding Docker's `CMD`).
-    * For MySQL 8+, you'll want to use the `mysql_native_password` password plugin.
+    * For MySQL 8+, you'll want to use the `mysql_native_password` authentication plugin
+        
+        Starting from MySQL 8.4, the `--default-authentication-plugin` above has been [depreciated](https://github.com/docker-library/mysql/issues/1048#issuecomment-2088836076). If you are using this verion, please make sure to revise the [process] section to use an updated syntax of setting mysql-native-password:
+        ```toml
+        [processes]
+          app = """--datadir /data/mysql \
+            --mysql-native-password=ON"""
+        ```
     * **Important**: For MySQL 5.7 and MySQL 8+, we set MySQL's data directory to a subdirectory of our mounted volume.
-
-If you're using MySQL 5.7, your `app` process can be simplified:
-
-```toml
-[processes]
-app = "--datadir /data/mysql"
-```
+      
+        If you're using MySQL 5.7, your `app` process can be simplified:
+        ```toml
+        [processes]
+        app = "--datadir /data/mysql"
+        ```
 
 <div class="important icon">
```

Comment:
Update this whole note:
```suggestion
<div class="important icon">
**Important**: Set MySQL's data directory to a subdirectory of your mounted volume. Mounting a disk in Linux usually results in a `lost+found` directory being created. However, MySQL won't initialize into a data directory unless it's completely empty. That's why you need to use a subdirectory of the mounted location: `/data/mysql`.
</div>
```

Reply:
Hi @andie787 !! Thank you for your reviews! I've updated the PR to reflect the suggestions you've made, it's a lot readable now, thank you!